src/main/java/com/airlines/entity/CrewMember.java

```diff
@@ -0,0 +1,84 @@
+package com.airlines.entity;
+
+import java.time.LocalDate;
+
+public class CrewMember {
+    private final int id;
+    private final String firsName;
+    private final String lastName;
+    private final String position;
+    private final LocalDate birthday;
+    private final String citizenship;
```

Comment:
You should use enum type here to make your code more safe

Reply:
fixed in 8d230f9c3c3300d7da8f899a28eb4137f3bf8074